pacemaker.spec.in

```diff
@@ -323,12 +324,12 @@ Requires:      %{name}-libs = %{version}-%{release}
 # systemd python bindings are separate package in some distros
 %if %{defined systemd_requires}
 
-%if 0%{?fedora} > 20
+%if 0%{?fedora} > 22
+Requires:      python2-systemd
```

Comment:
Not at this time in Fedora, there's just hard mapping:
- `python2-systemd` <-> `systemd-python`
- `python3-systemd` <-> `systemd-python3`
(canonical name <-> legacy alias [provided+obsoleted])
The change is mostly to be ready for when/if the legacy alias gets dropped.